ethclient/ethclient.go

```diff
@@ -781,3 +693,55 @@ func toCallArg(msg ethereum.CallMsg) interface{} {
 	}
 	return arg
 }
+
+// rpcProgress is a copy of SyncProgress with hex-encoded fields.
+type rpcProgress struct {
```

Comment:
IMO If removing these is not an option, we should at least keep them commented out

Reply:
Eh disagree, that'll make upgrading geth more complicated and the diffs with geth would be harder to see (if we keep on commenting things around). It doesn't hurt to have unused symbols in the code I would say

Reply:
Keeping it as is since it's unexported